src/pudl/transform/eiaaeo.py

```diff
@@ -193,9 +291,14 @@ def _decumulate(df: pd.DataFrame, decumulate_columns: list[str]) -> pd.DataFrame
         .pipe(_drop_aggregated_generation_data)
         .pipe(_collect_totals)
     )
+    # 1 BKWh = 1e12 Wh = 1e6 MWh
+    sanitized.loc[sanitized.units == "bkwh", "value"] *= 1e6
```

Comment:
The existing `convert_units` method seems more tailored to wide tables, and we're converting in the narrow-table format here, so I'm keeping it like this for now. Probably worth pulling out into a separate function at some point, and maybe putting some default conversions into the `filter_enrich_sanitize` method.